app/src/main/java/org/p2p/wallet/utils/DateTimeUtils.kt

```diff
@@ -68,6 +69,20 @@ object DateTimeUtils {
         }
     }
 
+    fun getDateRelatedFormatted(datetime: Long, context: Context): String {
+        val currentTime = System.currentTimeMillis()
```

Comment:
can be done with Duration too, but ok

Reply:
Duration ? what how? show me =)

Reply:
System.currentTimeMillis().toDuration(DurationUnit.MILLISECONDS)
maybe you can find something useful in those methods